### DISCUSSION TITLE
Attaching a file to comment randomly wipes out previously entered text

### DISCUSSION THREAD

**Discussion by php4fan** [community/community#121727] [Projects and Issues]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

Steps to reproduce:
1) Go to some issue in a GitHub repository that accepts comments
2) Start writing a comment. A very long comment that takes you half an hour of work to write

Now here's a sequence of actions that will certainly trigger the issue on purpose, but I suspect it might happen in other ways:

3) **select all** the text in your comment.
4) **Click outside** of the textarea for typing the comment.

NOTE that at this stage, **as far as the user can tell, nothing appears to be selected** 

5) go somewhere else outside of the browser to prepare some file that you are going to attach  to your very carefully crafted comment
6) come back to the browser tab where you are writing the comment. Remember, the textarea is NOT focused and nothing appears to be selected. You can see the text you typed in the textarea. There's no way you remember that you selected all the text at some point (and even if you did, you would have every right to assume that it got de-selected when you clicked outside and it stopped being visualized as selected)
7) Click on **"Paste, drop or click to add files"**. Select a file to upload and confirm

Observed behavior:
**KABOOM!!** all the text that you had typed is GONE, replaced by the file you uploaded.
And there's not even any way to recover it. Ctrl+Z does nothing!

Expected behavior:
Do I need to explain?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by nour-eldein10** [comment]
Is that a problem oe answer Cuz i don't know 😂😂

**Comment by adamya-singh** [comment]
this behavior is def a problem and very annoying

**Comment by php4fan** [comment]
It's been almost two years and the issue still exist. F***ing pathetic.
